test/Microsoft.Sbom.Api.Tests/Config/ConfigSanitizerTests.cs

```diff
@@ -51,6 +53,8 @@ public void Initialize()
 
         mockAssemblyConfig = new Mock<IAssemblyConfig>();
 
+        isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
Nitpick: Personally I would move this up to line 31 since it's simple and doesn't interop with any other Initialization.